.github/workflows/ci.yaml

```
@@ -54,3 +55,5 @@ jobs:
     - name: Run rstcheck Sanity
       run: |
         python tests/sanity.py rstcheck
+  nox:
```

The above two jobs have an empty-line separator. It'd be wise to keep that style here.
```suggestion

  nox:
```